src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -1,9 +1,31 @@
+// Node modules.
 import React from 'react';
 import moment from 'moment';
-
+import download from 'downloadjs';
+// Relative imports.
 import * as customPropTypes from '../prop-types';
 
-export default function SearchResult({ form }) {
+const onDownloadClick = url => event => {
+  // Escape early if we're not on IE.
+  if (!navigator.msSaveBlob) {
+    return;
+  }
+
+  // Prevent browser default behavior.
+  event.preventDefault();
+
+  try {
+    // Attempt to download the file.
+    const request = download(url);
+
+    // If we aren't able to, resort to opening the download link in a new tab.
+    request.onerror = window.open(url, '_blank');
+  } catch (error) {
+    window.open(url, '_blank');
+  }
```

Comment:
Is the `onerror` and `catch` necessary? Just wonder if we could do -

```js
try { download(url); }
catch.(err) {  window.open(url, '_blank'); }
```

Also wondering if the `download` function returns a Promise or something? I'm guessing not since it uses `XMLHttpRequest`

Reply:
`onerror` and `catch` are necessary in this case. If the xmlHttpRequest fails, it doesn't throw an error and instead executes `.onerror`. If the `download` module fails for whatever reason, we need the `try catch` statement.

Reply:
The `download` function returns an xmlHttpRequest object